BackofficeBundle/Resources/views/Tree/tree-macros.html.twig

```diff
@@ -55,9 +55,10 @@
                     href="#"
                     data-url="{{ path('open_orchestra_backoffice_node_new', {'parentId': parentId}) }}"
                     data-type="node"
-                >
+                >   <span class="btn btn-success">
```

Comment:
In view mode, I've got ">   <span class="btn btn-success">" too. You should have :
>
<span ....